interfaces/reader_test.go

```diff
@@ -5,7 +5,9 @@ package interfaces
 
 import (
 	"bytes"
-	oldcsv "encoding/csv"
+
+	oldcsv "github.com/eltorocorp/go-csv"
+
 	"testing"
 
 	thiscsv "github.com/eltorocorp/go-csv"
```

Comment:
this line: `thiscsv "github.com/eltorocorp/go-csv"` can be deleted since there is already a reference to the csv above.